astroplan/core.py

```diff
@@ -2,6 +2,21 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
+from astropy.coordinates import (EarthLocation, Latitude, Longitude, SkyCoord,
```

Comment:
@bmorris3 – Do you understand this? https://travis-ci.org/astroplanners/astroplan/jobs/67568895#L372  (I don't)